packages/client/src/devices/BrowserPermission.ts

```diff
@@ -73,10 +73,16 @@ export class BrowserPermission {
           const isGranted = this.state === 'granted';
 
           if (!isGranted && throwOnNotAllowed) {
-            throw new DOMException(
-              'Permission was not granted previously, and prompting again is not allowed',
-              'NotAllowedError',
-            );
+            if (typeof DOMException !== 'undefined') {
+              throw new DOMException(
+                'Permission was not granted previously, and prompting again is not allowed',
+                'NotAllowedError',
+              );
+            } else {
+              throw new Error(
+                'Permission was not granted previously, and prompting again is not allowed',
+              );
+            }
```

Comment:
Does it matter if we throw a `DOMException` or a regular `Error`? Can we go with `throw new Error` and not have multiple code branches to achieve the same thing?

Reply:
@myandrienko can you clarify here, DOMException does something special on browsers?

Reply:
removed it, as per MDN, it was for exception thrown by a web API, so not really necessary here